package.json

```diff
@@ -11,6 +11,7 @@
     "node": ">=5"
   },
   "scripts": {
+    "postinstall": "cd lighthouse-core && npm install && npm run build",
```

Comment:
I added a `prepublish` script to all directories that need to compile things and also one in the root's package.json. `npm install` will run `npm run build-core && npm run build-cli && npm run build-viewer`